Packs/DeCYFIR/Integrations/decyfir/decyfir.py

```diff
@@ -0,0 +1,483 @@
+LABEL_EXECUTIVE_PEOPLE = "Executive People"
+LABEL_PRODUCT_SOLUTION = "Product Solution"
+LABEL_SOCIAL_HANDLERS = "Social Handlers"
+
+LABEL_PHISHING = "Phishing"
+LABEL_RANSOMWARE = "Ransomware"
+LABEL_DARK_WEB = "Dark web"
+
+LABEL_SOURCE_CODE = "Source Code"
+LABEL_MALICIOUS_MOBILE_APPS = "Malicious Mobile Apps"
+LABEL_CONFIDENTIAL_FILES = "Confidential Files"
+LABEL_DUMPS_PII_CII = "Dumps PII-CII"
+LABEL_SOCIAL_THREAT = "Social Threat"
+
+LABELS_LIST: list = [LABEL_OPEN_PORTS, LABEL_IP_VULNERABILITY, LABEL_CONFIGURATION, LABEL_CLOUD_WEAKNESS,
+                     LABEL_IP_REPUTATION, LABEL_CERTIFICATES,
+                     LABEL_DOMAIN_IT_ASSET, LABEL_EXECUTIVE_PEOPLE, LABEL_PRODUCT_SOLUTION, LABEL_SOCIAL_HANDLERS,
+                     LABEL_PHISHING, LABEL_RANSOMWARE, LABEL_DARK_WEB,
+                     LABEL_SOURCE_CODE, LABEL_MALICIOUS_MOBILE_APPS, LABEL_CONFIDENTIAL_FILES, LABEL_DUMPS_PII_CII,
+                     LABEL_SOCIAL_THREAT]
+
+VAR_ATTACK_SURFACE = "attack-surface"
+VAR_IMPERSONATION_AND_INFRINGEMENT = "impersonation-and-infringement"
+VAR_DATA_BREACH_AND_WEB_MONITORING = "data-breach-and-web-monitoring"
+VAR_SOCIAL_AND_PUBLIC_EXPOSURE = "social-and-public-exposure"
+
+# ATTACK SURFACE
+VAR_OPEN_PORTS = "open-ports"
+VAR_IP_VULNERABILITY = "ip-vulnerability"
+VAR_CONFIGURATION = "configuration"
+VAR_CLOUD_WEAKNESS = "cloud-weakness"
+VAR_IP_REPUTATION = "ip-reputation"
+VAR_CERTIFICATES = "certificates"
+
+VAR_ATTACK_SURFACES_SUB_TYPES: list = [VAR_OPEN_PORTS, VAR_IP_VULNERABILITY, VAR_CONFIGURATION, VAR_CLOUD_WEAKNESS,
+                                       VAR_IP_REPUTATION, VAR_CERTIFICATES]
+
+# IMPERSONATION & INFRINGEMENT
+VAR_DOMAIN_IT_ASSET = "domain-it-asset"
+VAR_EXECUTIVE_PEOPLE = "executive-people"
+VAR_PRODUCT_SOLUTION = "product-solution"
+VAR_SOCIAL_HANDLERS = "social-handlers"
+
+VAR_IMPERSONATION_AND_INFRINGEMENT_SUB_TYPE: list = [VAR_DOMAIN_IT_ASSET, VAR_EXECUTIVE_PEOPLE, VAR_PRODUCT_SOLUTION,
+                                                     VAR_SOCIAL_HANDLERS]
+
+# DATA BREACH AND WEB MONITORING
+VAR_PHISHING = "phishing"
+VAR_RANSOMWARE = "ransomware"
+VAR_DARK_WEB = "dark-web"
+
+VAR_DATA_BREACH_AND_WEB_MONITORING_SUB_TYPES: list = [VAR_PHISHING, VAR_RANSOMWARE, VAR_DARK_WEB]
+
+# SOCIAL AND PUBLIC EXPOSURE
+VAR_SOURCE_CODE = "source-code"
+VAR_MALICIOUS_MOBILE_APPS = "malicious-mobile-apps"
+VAR_CONFIDENTIAL_FILES = "confidential-files"
+VAR_DUMPS_PII_CII = "dumps-pii-cii"
+VAR_SOCIAL_THREAT = "social-threat"
+
+VAR_SOCIAL_AND_PUBLIC_EXPOSURE_SUB_TYPES: list = [VAR_SOURCE_CODE, VAR_MALICIOUS_MOBILE_APPS, VAR_CONFIDENTIAL_FILES,
+                                                  VAR_DUMPS_PII_CII, VAR_SOCIAL_THREAT]
+
+
+class Client(BaseClient):
+    """
+    Client will implement the service API, and should not contain any Demisto logic.
+    Should only do requests and return data.
+    """
+
+    def say_hello(self, name):
+        return f'Hello {name}'
```

Comment:
Please remove.

Reply:
We corrected this.  we can push next final commit.